backend/level-3/exercise.spec.ts

```diff
@@ -43,45 +45,64 @@ interface IContract {
  * Define the contract schema and model that will be saved into MongoDB.
  * NOTE: make sure to use the interface above! 
  */
-const contractSchema = new Schema<unknown>({
-    // TODO: add here
+const contractSchema = new Schema<IContract>({
+    address: String,
+    name: String,
+    abi: [{ type: Schema.Types.Mixed }]
 })
 
-const Contract = model<unknown>('unknown', contractSchema)
+const Contract = model<IContract>('Contract', contractSchema)
 
 /** Exercise 3:
  * 
  * Define the following APIs:
  * - GET /: returns all the contracts available
  * - GET /:address : returnss the contract with the given address, returns 404 if not found
  */
-app.get('/', (req: unknown, res: unknown) => {
-    // TODO: complete this function
+app.get('/', async (req: unknown, res: unknown) => {
+    const myRes = res as express.Response
+    try{
+        const contracts = await Contract.find()
+        myRes.send(contracts)
+    } catch(error) {
+        myRes.status(500).send(error)
+    }
 })
 
-
-app.get('/:address', (req: unknown, res: unknown) => {
-    // TODO: complete this function
+app.get('/:address', async (req: unknown, res: unknown) => {
+    const myRes = res as express.Response
+    const myReq = req as express.Request
```

Comment:
This is wrong. Think of another way to define this types without using `as`.

Tip: you can update everything except the tests